template cleanup and start of README (#20)
opencasestudies/ocs-template-quarto: README.md

# Bio-OCS Quarto Template

> This is a template for creating websites from qmd files hosted on GitHub with three helpful automations following a pull request to the repository: spelling check, broken link check, and website rendering. The spelling and url checks as well as the website rendering are supported by the OTTR Template, specifically the OTTR Quarto Web version.

Reviewer: ```suggestion
This is a template for creating websites from qmd files hosted on GitHub with three helpful automations following a pull request to the repository: 

- Spelling check
- Broken link check
- Website rendering.

This is all supported by the OTTR Template, specifically the [OTTR Quarto Web](https://github.com/ottrproject/OTTR_Quarto_Website) version.
```

The rest of these files will largely not need updated though a brief description of what each contains is provided.

* styles.css -- Defines html elements
* _brand.yml -- Logo and favicon definition
* config_automation.yml	-- Configurations for the OTTR Template automations. Update this file to toggle spelling or url check as well as minimum errors allowed for those checks or to update the docker image.

### Directories

* `data/`: Raw, imported, and wrangled datasets. `raw/pm25_data.csv`, `imported/pm25_data_imported.rda`, and `wrangled_data.csv` are template files that can be removed or overwritten by case study repositories.
* `img/`: Template images. Please do not add case study images in this directory -- instead adding them to `resources` or `resources/images`.
* `resources/`: Files related to OTTR checks (e.g., dictionary and a file of URLs to exclude/ignore from checking.). Also should put template specific images here.
* `.github/`: workflow files supporting the OTTR template automations
* `docs/`: the rendered files that GitHub pages will use -- refreshed through OTTR template automations.

### Content

> For the overall content of the Open Case Study, each portion of the case study is split into individual `.qmd` files. They are included in order within the `index.qmd` file. Certain case studies will not have every component, and so we can toggle those on and off using short code and header booleans. Below is a list of all of the sub-component `.qmd` files in checklist form. Besides the listed changes below, `index.qmd` should not need updated.

Reviewer: ```suggestion
For the overall content of the Open Case Study, each portion of the case study is split into individual `.qmd` files. They are included in order within the `index.qmd` file. Certain case studies will not have every component, and so we can toggle those on and off using short code and header booleans.  This is mostly for others using our case studies to teach, as they might want to have a version that only has part of the case study.  For our core team, we will likely have all components for our case studies. Below is a list of all of the sub-component `.qmd` files in checklist form. Besides the listed changes below, `index.qmd` should not need updated.
```
